.travis.yml

```diff
@@ -45,6 +45,7 @@ install:
 
   - if [[ $TRAVIS_PYTHON_VERSION == '2.7' || $TRAVIS_PYTHON_VERSION == '3.4' ]]; then conda install spark=$SPARK_VERSION -c blaze -c anaconda-cluster; fi
   - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda install pyhive -c blaze; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then pip install future; fi
```

Comment:
in py2 slice is in `__builtin__`, instead of adding a new package can we just put slice in compat?